js/summary.go

```diff
@@ -36,10 +36,12 @@ import (
 // TODO: move this to a separate JS file and use go.rice to embed it
 const summaryWrapperLambdaCode = `
 (function() {
-	var forEach = function(obj, callback) {
+	var forEach = function (obj, callback) {
 		for (var key in obj) {
 			if (obj.hasOwnProperty(key)) {
-				callback(key, obj[key]);
+				if (callback(key, obj[key])) {
+					break;
+				}
```

Comment:
Why was this added? I don't see any of the callbacks returning `true`.

Would it make more sense to `catch` an error to trigger this `break` instead?

Reply:
It was added for the text summary generation code. It currently lives in https://gist.github.com/na--/eaf6540fae765499cbfddce1a052583f, but I'll clean it up and push it to jslib.k6.io soon, and it will be bundled in the k6 v0.31.0 cycle, so the `forEach` changes will soon be used. And anyway, returning a result seems cleaner than throwing an error - for one thing, I wouldn't want to catch other actual errors by mistake.